src/ui/components/LeagueTopBar.tsx

```diff
@@ -81,24 +115,20 @@ const LeagueTopBar = memo(() => {
 	if (show) {
 		// Show only the first upcoming game
 		for (const game of prevGames.current) {
-			games2.push(game);
+			games2.unshift(game);
 			if (game.teams[0].pts === undefined) {
 				break;
 			}
 		}
-
-		const start = games2.length - numberOfScoreBoxes;
-		if (start > 0) {
-			games2 = games2.slice(start);
-		}
 	}
 
 	const transition = { duration: 0.2, type: "tween" };
 
 	return (
 		<div
-			className="league-top-bar flex-shrink-0 d-flex justify-content-end overflow-hidden mt-2"
+			className="league-top-bar flex-shrink-0 d-flex overflow-auto flex-row-reverse pr-3 pl-1 pb-1 mt-2"
```

Comment:
Also added pl-1 to give some space on the left side when scrolled all the way to the left end